monitoring/grafana/dashboards/tox.ini

```
@@ -20,3 +20,14 @@ passenv = GRAFONNET_PATH
 commands =
      check: bash test-jsonnet.sh
      fix: jsonnet -m . jsonnet/grafana_dashboards.jsonnet
+
+[testenv:grafonnet-{query_test}]
```

```suggestion
[testenv:grafonnet-{query-test}]
```
Can we have consistency in naming: either hyphens or underscores?
IMHO a mix of both is harder to type...